### ISSUE TITLE
pgvector for vector store/retrieve

### ISSUE THREAD

**Issue by aldarisbm** [aldarisbm/memory#3]
https://github.com/pgvector/pgvector

**Comment by d0rc** [comment]
yes, this one would be nice, as well as any other open-source / self-hosted solution,
I hope we'll have a chance to discuss it
